tools/deb/conffiles

```diff
@@ -0,0 +1,3 @@
+/var/lib/istio/envoy/xsidecar.env
```

Comment:
extra x ?

Reply:
Sorry, used it for debugging (to not override my previous config). 

No, it doesn't offer to merge - asks user if he wants to override. 
An option would be to provide a sidecar.env.sample or none at all - the provisioning 
tool will create it. 

But one step at a time - right now the critical problem is that sidecar.env is replaced on
upgrade losing any config, and that's the standard .deb solution.